src/syntax/parser.rs

```diff
@@ -2022,6 +2022,106 @@ impl<'a> Parser<'a> {
         })
     }
 
+    fn parser_import_specification(&mut self) -> Result<ImportSpecification> {
+        let imported = self.parse_identifier_name()?;
+        let local = if self.match_contextual_keyword(interner::RESERVED_AS) {
+            self.scanner.next_token()?;
+            self.parse_identifier_name()?
+        } else {
+            imported.clone()
```

Comment:
Could make `local` optional so this could be `None` instead of cloning.

I'm not sure if it's worth it though.. cloning `Id` is dirt cheap if I recall correctly

Reply:
Cloning `Id` should be something like copying 5-6 `u32`s , so peanuts.